docs/api/structures/window-open-handler-response.md

```diff
@@ -0,0 +1,22 @@
+# WindowOpenHandlerResponse Object
+
+* `action` string - Can be `allow` or `deny`. Controls whether new window should be created.
+* `overrideBrowserWindowOptions` BrowserWindowConstructorOptions (optional) - Allows customization of the created window.
+* `outlivesOpener` boolean (optional) - Controls whether created window should stay open when its opener was closed.
+* `createWindow` (options: BrowserWindowConstructorOptions) => BrowserWindow (optional) - If specified, will be called instead of `new BrowserWindow` to create the new child window. This can be used for example to have the new window open as a BrowserView instead of in a separate window.
```

Comment:
I think this should probably return a `WebContents` instead of a `BrowserWindow`. For example, it should be possible to open a child window as a BrowserView inside an existing BrowserWindow. I don't think the calling code requires `BrowserWindow`, since all it does is call `loadURL` I think.

Reply:
`openGuestWindow` which uses this function expects to return a `BrowserWindow | undefined` but nothing is using this return value so I guess returning `undefined` in such case is okay?

Reply:
@SEUH WebContents is the right thing to return here, as that's what the calling function needs. I could see it being renamed `createWebContents` but that isn't quite the right name either, because in some cases it will not create a new WebContents, for example when the caller provides a `webContents` property in the BrowserWindowConstructorOptions, it should be used to create the new window.